compiler/src/iree/compiler/Codegen/Common/test/generic_vectorization.mlir

```diff
@@ -366,3 +367,48 @@ func.func @generic_unpack_infer_vector_size(%arg0: tensor<?x?x16x16xf32>, %arg1:
 // CHECK-MASK:           %[[GENERIC_SRC:.+]] = vector.transfer_read %[[UNPACK_WRITE]]{{.+}}, %[[GENERIC_MASK]]
 // CHECK-MASK:           %[[EXP:.+]] = math.exp %[[GENERIC_SRC]]
 // CHECK-MASK:           vector.transfer_write %[[EXP]]{{.+}}, %[[GENERIC_MASK]]
+
+// -----
+
+#aarch64_sve = #hal.executable.target<"llvm-cpu", "embedded-elf-arm_64", {cpu_features = "+sve", target_triple = "aarch64-none-elf"}>
+#map = affine_map<()[s0] -> (-(176 mod s0) + 176)>
+
+hal.executable private @test {
+  hal.executable.variant public @test target(#aarch64_sve) {
+    builtin.module {
+      func.func @dynamic_fill_with_scalable_tiling_infer_vector_size(%arg0: tensor<1x67x120x176xf32>) -> tensor<1x67x120x176xf32> {
```

Comment:
You can put the target attribute to `func.func`, then we don't need to create a new check config. E.g.,

https://github.com/iree-org/iree/blob/2eea916a777947ef4162a2aa44f47a2c6c1fb926/compiler/src/iree/compiler/Codegen/Common/CPU/test/llvmcpu_materialize_encoding.mlir#L4-L6

Reply:
Much nicer, thanks!